templates/flake.nix

```diff
@@ -0,0 +1,57 @@
+{
+  description = "Rust-Nix";
+
+  inputs = {
+    nixpkgs.url = "github:nixos/nixpkgs/nixos-unstable";
+    flake-parts.url = "github:hercules-ci/flake-parts";
+    rust-overlay.url = "github:oxalica/rust-overlay";
+    rust-crate2nix = {
+      url = "github:kolloch/crate2nix";
+      flake = false;
+    };
+  };
+
+  outputs = inputs @ {
+    self,
+    nixpkgs,
+    flake-parts,
+    rust-overlay,
+    rust-crate2nix,
+    ...
+  } : flake-parts.lib.mkFlake { inherit inputs; } {
+    flake = {
+    };
+    systems = [
+      "x86_64-linux"
+      "x86_64-darwin"
```

Comment:
Since it is a template, it is maybe Ok but I guess aarch64 for darwin is more comon nowadays?

Reply:
done